### ISSUE TITLE
Button component template

### ISSUE THREAD

**Issue by ponnexcodev** [Toniq-Labs/toniq-labs-design-system#21]
The current toniq button is used as follow `<ToniqButton text="Buy Now" />`

There are also buttons in the design that don't use text but rather icons.
![Screen Shot 2022-05-13 at 10 32 07 AM](https://user-images.githubusercontent.com/99699764/168199931-726b5399-4bad-4471-827b-1f82fcac9c44.png)

Let's make as template: `<ToniqButton>Buy Now</ToniqButton>`

use other html:
```
<ToniqButton>
  <div>
    <span>Buy Now</span>
  </div>
</ToniqButton>
```

and maybe use other components as well
```
<ToniqButton>
  <ToniqIcon title="iconName" icon="icon"/>
</ToniqButton>
```

**Comment by electrovir** [comment]
I was thinking maybe we could simply add an `icon` input to `ToniqButton` rather than opening up the button to _any_ child content? Then the button component itself can handle creating an instance of `ToniqIcon`.

**Comment by ponnexcodev** [comment]
I was thinking the same as well, there are also buttons that will be a mix of both text and icon as well as other future uses. And doing so in the code above makes it much easier to use rather than adding other properties from time to time.

**Comment by electrovir** [comment]
Merged in #20
